pkg/util/util.go

```diff
@@ -20,6 +20,15 @@ import (
 	"io"
 )
 
+// Encode writes data of specified interfaces to io writer
+func Encode(buff io.Writer, inputs ...interface{}) error {
+	var err error
+	for _, in := range inputs {
+		err = binary.Write(buff, binary.BigEndian, in)
```

Comment:
have endianness as paramter in both encode and decode functions to increase their utility

Reply:
Makes sense. Thanks